<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgProducts</class>
 <widget class="QWidget" name="wdgProducts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>263</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Products list</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Select your search</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Search by code, ISIN, ticker or product name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="chkStockExchange">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Filter by stock exchange</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cmbStockExchange">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QToolButton" name="cmd">
            <property name="icon">
             <iconset resource="../images/xulpymoney.qrc">
              <normaloff>:/xulpymoney/search.png</normaloff>:/xulpymoney/search.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="mqtwObjects" name="mqtwInvestments" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="lblFound">
       <property name="text">
        <string>Registers found</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionProductReport">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/books.png</normaloff>:/xulpymoney/books.png</iconset>
   </property>
   <property name="text">
    <string>Product report</string>
   </property>
   <property name="toolTip">
    <string>Product report</string>
   </property>
  </action>
  <action name="actionIbex35">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/countries/spain.gif</normaloff>:/countries/spain.gif</iconset>
   </property>
   <property name="text">
    <string>Ibex 35</string>
   </property>
  </action>
  <action name="actionProductNew">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>New product</string>
   </property>
   <property name="toolTip">
    <string>New user product</string>
   </property>
  </action>
  <action name="actionProductDelete">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/list-remove.png</normaloff>:/xulpymoney/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete product</string>
   </property>
   <property name="toolTip">
    <string>Delete user product</string>
   </property>
  </action>
  <action name="actionFavorites">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/star.png</normaloff>:/xulpymoney/star.png</iconset>
   </property>
   <property name="text">
    <string>Add to favorites</string>
   </property>
   <property name="toolTip">
    <string>Add to favorites</string>
   </property>
  </action>
  <action name="actionMergeCodes">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/merge.png</normaloff>:/xulpymoney/merge.png</iconset>
   </property>
   <property name="text">
    <string>Merge selected codes</string>
   </property>
   <property name="toolTip">
    <string>Merge selected codes</string>
   </property>
  </action>
  <action name="actionQuoteNew">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>New price</string>
   </property>
   <property name="toolTip">
    <string>New price</string>
   </property>
  </action>
  <action name="actionEstimationDPSNew">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>New DPS estimation</string>
   </property>
   <property name="toolTip">
    <string>New Dividend per share estimation</string>
   </property>
  </action>
  <action name="actionPurge">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/tools-wizard.png</normaloff>:/xulpymoney/tools-wizard.png</iconset>
   </property>
   <property name="text">
    <string>Purge investment</string>
   </property>
   <property name="toolTip">
    <string>Deletes quotes innecesary. Leaves open, high, low and close quotes.</string>
   </property>
  </action>
  <action name="actionPurchaseGraphic">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/report.png</normaloff>:/xulpymoney/report.png</iconset>
   </property>
   <property name="text">
    <string>Show purchase graphic</string>
   </property>
   <property name="toolTip">
    <string>Show purchase graphic</string>
   </property>
  </action>
  <action name="actionProductPriceLastRemove">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/list-remove.png</normaloff>:/xulpymoney/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove last product price</string>
   </property>
   <property name="toolTip">
    <string>Remove last product price</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mqtwObjects</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txt</tabstop>
 </tabstops>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chkStockExchange</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbStockExchange</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
